control-plane/grpc/src/operations/volume/traits_snapshots.rs

```diff
@@ -132,24 +134,74 @@ pub struct ReplicaSnapshot {
     source_id: ReplicaId,
 }
 
+/// Volume replica snapshot state information.
+#[derive(Debug)]
+#[allow(unused)]
+pub enum VolumeReplicaSnapshotState {
+    /// When the snapshot is available.
+    Online {
+        /// The pool hosting the snapshot.
+        pool_id: PoolId,
+        /// The snapshot state.
```

Comment:
`The snapshot state.` --> `The replica snapshot state.` Just to be exactly clear while hovering in IDEs. Same in below?

Reply:
Indeed, done